src/app/zap-templates/zcl/data-model/chip/operational-state-cluster.xml

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
   <enum name="OperationalStateEnum" type="enum8">
     <cluster code="0x0060"/>
+    <cluster code="0x0048"/>
```

Comment:
src/app/zap-templates/zcl/data-model/chip/operational-state-rvc-cluster.xml does not re-use this enum and instead defines it's own with extra values from the spec. I am inclined to think we should do the same here? 

Any reason to re-use this generic one?

Reply:
We are not adding new states to the enum per https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/8349. We are utilizing only what is in the base Operational State cluster

Reply:
This file is no longer part of this PR. This change has already been merged to the master.